scripts/release-test-ci.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Stop in case of error
+set -e
+
+# To be run as part of a release test only on Linux
+# requires python, pytest, xdist; install e.g. via
+# sudo apt install python3 python3-pytest python3-pytest-xdist python3-psutil
+
+# must be run in main folder
+# multicore machine recommended for fast execution
+
+# expect (ideally latest/release-test) liboqs to be already build and present
+if [ -d liboqs ]; then
+   export LIBOQS_SRC_DIR=`pwd`/liboqs
+else
+   echo "liboqs not found. Exiting."
+   exit 1
+fi
+
+if [ -d oqs-template ]; then
+    # Activate all algorithms
+    sed -i "s/enable\: false/enable\: true/g" oqs-template/generate.yml
+    python3 oqs-template/generate.py
+    ./scripts/fullbuild.sh
```

Comment:
As this adds a specific CI-executed test script, I'd suggest forcing it to run on the latest OpenSSL code. All that'd need is setting the env var "OPENSSL_BRANCH=master" for `fullbuild.sh`.

Reply:
Since the local wrapper triggers this script as well, I'll pass the environment variable in the CI config. That way it can still be set to a different value in the local script.